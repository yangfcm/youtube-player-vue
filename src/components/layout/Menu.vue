<template>
  <div class="ui three item menu red stackable">
    <router-link class="item" to="/" exact active-class="active">
      <i class="thumbs up outline icon"></i> Recommend
    </router-link>
    <router-link class="item" to="/subscription" active-class="active">
      <i class="user plus icon"></i> Subscription
    </router-link>
    <router-link class="item" to="/playlist" active-class="active"
      ><i class="list icon"></i> Playlist</router-link
    >
  </div>
</template>

<script>
export default {};
</script>

<style scoped></style>
