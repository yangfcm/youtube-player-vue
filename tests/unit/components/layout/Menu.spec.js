import { shallowMount, RouterLinkStub } from "@vue/test-utils";
import Menu from "@/components/layout/Menu.vue";

test("Menu component can render correctly", () => {
  const wrapper = shallowMount(Menu, {
    stubs: {
      RouterLink: RouterLinkStub,
    },
  });
  expect(wrapper).toMatchSnapshot();
});
