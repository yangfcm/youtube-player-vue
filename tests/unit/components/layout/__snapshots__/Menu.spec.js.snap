// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component can render correctly 1`] = `
<div class="ui three item menu red stackable"><a class="item"><i class="thumbs up outline icon"></i> Recommend
  </a> <a class="item"><i class="user plus icon"></i> Subscription
  </a> <a class="item"><i class="list icon"></i> Playlist</a></div>
`;
